src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java

```diff
@@ -30,6 +30,9 @@
   static String oracleId;
   static String queryId;
 
+  static String queryString = "Am I stupid?";
+  static String responseString = "yes you are nuts!";
```

Comment:
please use a more senseful test string

Reply:
now using an example query and response from https://api.openweathermap.org/